Sources/BaggageContext/Context.swift

```diff
@@ -0,0 +1,291 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Baggage Context open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Swift Baggage Context project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@_exported import Baggage
```

Comment:
👍

Reply:
Trying to check if we're okey to do that or if we shouldn't